omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.controller.js

```diff
@@ -55,7 +55,7 @@ export default class ObsAcrossEncountersController {
 
   fetchEncounters() {
     const encounterTypes = this.config.encounterTypes ? this.config.encounterTypes.split(',').map(c => c.trim()) : [];
-    const legacyEncounterTypes = this.config.encounterType ? this.config.encounterType.split(',').map(c => c.trim()) : [];
+    const legacyEncounterTypes = this.config.encounterType ? this.config.encounterType.split(',').map(c => c.trim()) : ["123AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"];
```

Comment:
Does this mean that when an encounter type is not provided, we use a hard coded one?